cmd/swapd/signal_handler.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
This isn't a new file.  I renamed `wait.go` to `signal_handler.go` and made some updates.